src/routes/+page.svelte

```diff
@@ -1,6 +1,40 @@
-<div class="flex flex-col items-center justify-evenly h-dvh text-text_white">
-	<h1 class="text-center text-5xl font-bold">hiiii :3</h1>
-	<a class="text-center text-2xl rounded bg-btn_grey px-4 py-2 font-bold" href="/scout">
-		scout
-	</a>
+<script lang="ts">
+	import { browser } from '$app/environment';
+	import { goto } from '$app/navigation';
+	import { onMount } from 'svelte';
+	let name: string = (browser && window.localStorage.getItem('name')) || '';
+	let inputname: string;
+	onMount(() => {
+		let isLoggedIn = browser != null && window.localStorage.getItem('name') != '';
+		if (isLoggedIn) {
+			goto('/homepage');
+		}
+	});
+	function login() {
+		if (inputname === '') {
+			alert('Enter your name');
```

Comment:
generally alerts aren't used, and in this case it's defiantly redundant b/c the placeholder text already provides that info to the user